perfmetrics/scripts/presubmit_test/git_pr_test/presubmit.cfg

```diff
@@ -0,0 +1 @@
+build_file: "gcsfuse/perfmetrics/scripts/presubmit_test/git_pr_test/build.sh"
```

Comment:
should we call it pr_perf_test instead of git_pr_test

Reply:
fixed